microsoft/testsuites/performance/common.py

```diff
@@ -40,11 +38,11 @@ def perf_disk(
     start_iodepth: int,
     max_iodepth: int,
     filename: str,
-    core_count: int,
-    disk_count: int,
-    disk_setup_type: DiskSetupType,
-    disk_type: DiskType,
     test_result: TestResult,
+    core_count: int = 0,
+    disk_count: int = 0,
+    disk_setup_type: Any = "",
+    disk_type: Any = "",
```

Comment:
Why the change in data types and ordering of the parameters?

Reply:
Datatype were related to specific to already implemented testcases. This is not needed by our generic testcase of FIO and these args are getting used to stamp with notifier message only.  So, We have to make it default.

Order has to be changed as default args cannot be written before mandatory args.

Reply:
Why not use `Optional` instead of changing type to `Any`?

Reply:
They are metrics, which are collected to database. So it's better to fill them in to get data more accurate.